client/my-sites/stats/activity-log/index.jsx

```diff
@@ -493,7 +493,8 @@ export default connect(
 		const requestedBackupId = getRequestedBackup( state, siteId );
 		const rewindState = getRewindState( state, siteId );
 		const restoreStatus = rewindState.rewind && rewindState.rewind.status;
-		const logs = siteId && requestActivityLogs( siteId, {} );
+		const filter = siteId && getActivityLogFilter( state, siteId );
```

Comment:
we shouldn't have had the `siteId` check here as it's already handled in the selector - I may have suggested we add it in, sorry if I did

this introduces an error when no `siteId` exists which is `filter.page` because `filter` doesn't exist

Reply:
yep, i noticed that too. i had a different solution baked into https://github.com/Automattic/wp-calypso/pull/25902 

I think just removing `siteId` check should be fine